www-apps/prewikka/prewikka-5.2.0.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+PYTHON_REQ_USE="ssl"
+
+inherit distutils-r1 optfeature
+
+DESCRIPTION="Graphical front-end analysis console for the Prelude Framework"
+HOMEPAGE="https://www.prelude-siem.org"
+SRC_URI="https://www.prelude-siem.org/pkg/src/5.2.0/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND=">=dev-libs/libprelude-5.2.0[python,${PYTHON_USEDEP}]
+	<dev-libs/libprelude-6[python,${PYTHON_USEDEP}]
+	>=dev-libs/libpreludedb-5.2.0[python,${PYTHON_USEDEP}]
+	<dev-libs/libpreludedb-6[python,${PYTHON_USEDEP}]
+	dev-python/Babel[${PYTHON_USEDEP}]
+	dev-python/croniter[${PYTHON_USEDEP}]
+	dev-python/gevent[${PYTHON_USEDEP}]
+	dev-python/lark-parser[${PYTHON_USEDEP}]
+	dev-python/mako[${PYTHON_USEDEP}]
+	dev-python/python-dateutil[${PYTHON_USEDEP}]
+	dev-python/pytz[${PYTHON_USEDEP}]
+	dev-python/pyyaml[${PYTHON_USEDEP}]
+	dev-python/voluptuous[${PYTHON_USEDEP}]
+	dev-python/werkzeug[${PYTHON_USEDEP}]
+	media-fonts/font-xfree86-type1"
+
+DEPEND="${RDEPEND}"
+
+BDEPEND="dev-python/lesscpy[${PYTHON_USEDEP}]
+	sys-devel/gettext"
+
+PATCHES=(
+	"${FILESDIR}/${PN}-5.2.0-fix_siteconfig.patch"
```

Comment:
Looks like you forgot to push this patch, 
```
 * ERROR: www-apps/prewikka-5.2.0::testworld failed (prepare phase):
 *   patch -p1  failed with /var/tmp/portage/www-apps/prewikka-5.2.0/files/prewikka-5.2.0-fix_siteconfig.patch
 * 
 * Call stack:
 *               ebuild.sh, line  125:  Called src_prepare
 *             environment, line 2951:  Called distutils-r1_src_prepare
 *             environment, line 1214:  Called distutils-r1_python_prepare_all
 *             environment, line 1121:  Called default
 *      phase-functions.sh, line  855:  Called default_src_prepare
 *      phase-functions.sh, line  920:  Called __eapi6_src_prepare
 *             environment, line  218:  Called eapply '/var/tmp/portage/www-apps/prewikka-5.2.0/files/prewikka-5.2.0-fix_siteconfig.patch'
 *             environment, line 1482:  Called _eapply_patch '/var/tmp/portage/www-apps/prewikka-5.2.0/files/prewikka-5.2.0-fix_siteconfig.patch'
 *             environment, line 1420:  Called __helpers_die 'patch -p1  failed with /var/tmp/portage/www-apps/prewikka-5.2.0/files/prewikka-5.2.0-fix_siteconfig.patch'
 *   isolated-functions.sh, line  112:  Called die
 * The specific snippet of code:
 *   		die "$@"
 * 
 * If you need support, post the output of `emerge --info '=www-apps/prewikka-5.2.0::testworld'`,
 * the complete build log and the output of `emerge -pqv '=www-apps/prewikka-5.2.0::testworld'`.
 * The complete build log is located at '/var/tmp/portage/www-apps/prewikka-5.2.0/temp/build.log'.
 * The ebuild environment file is located at '/var/tmp/portage/www-apps/prewikka-5.2.0/temp/environment'.
 * Working directory: '/var/tmp/portage/www-apps/prewikka-5.2.0/work/prewikka-5.2.0'
 * S: '/var/tmp/portage/www-apps/prewikka-5.2.0/work/prewikka-5.2.0'

>>> Failed to emerge www-apps/prewikka-5.2.0, Log file:
```

Reply:
done